ddtrace/appsec/_iast/_ast/visitor.py

```diff
@@ -671,17 +672,15 @@ def visit_Subscript(self, subscr_node):  # type: (ast.Subscript) -> Any
         )
         if isinstance(subscr_node.slice, ast.Slice):
             # Slice[0:1:2]. The other cases in this if are Indexes[0]
-            # TODO: Add stlice
-            # aspect_split = self._aspect_slice.split(".")
-            # call_node.func.attr = aspect_split[1]
-            # call_node.func.value.id = aspect_split[0]
-            # none_node = self._none_constant(subscr_node)
-            # lower = subscr_node.slice.lower if subscr_node.slice.lower is not None else none_node
-            # upper = subscr_node.slice.upper if subscr_node.slice.upper is not None else none_node
-            # step = subscr_node.slice.step if subscr_node.slice.step is not None else none_node
-            # call_node.args.extend([subscr_node.value, lower, upper, step])
-            # self.ast_modified = True
-            return subscr_node
+            aspect_split = self._aspect_slice.split(".")
+            call_node.func.attr = aspect_split[1]
+            call_node.func.value.id = aspect_split[0]
+            none_node = self._none_constant(subscr_node)
+            lower = subscr_node.slice.lower if subscr_node.slice.lower is not None else none_node
+            upper = subscr_node.slice.upper if subscr_node.slice.upper is not None else none_node
+            step = subscr_node.slice.step if subscr_node.slice.step is not None else none_node
```

Comment:
```suggestion
            lower = none_node if subscr_node.slice.lower is None else subscr_node.slice.lower
            upper = none_node if subscr_node.slice.upper is None else subscr_node.slice.upper 
            step = none_node if subscr_node.slice.step is None else subscr_node.slice.step
```
Just to avoid double negation logic